roles/cluster_upgrade/tasks/main.yml

```diff
@@ -164,47 +225,59 @@
             scheduling_policy_properties: "{{ cluster_scheduling_policy_properties }}"
           when: use_maintenance_policy and cluster_policy.changed | default(false)
 
+        - name: progress 95% - host upgrades are done (successful or not), only need to start VMs that were stopped by the playbook
+          include_tasks: log_progress.yml
+          vars:
+            progress: 95
+            description: "host upgrades are done (successful or not), restarting non-migratable VMs"
+
+        # TODO: These VMs aren't explicity stopped anywhere...should they be?
```

Comment:
If I remember correctly the VMs are stopped when doing the host upgrade by themselves so I think we should remove the parameter `stopped_vms` it has not been in use in this role for multiple years. We handle here only non-migratable VMs

Reply:
Since even the cluster upgrade wizard does not use that option, I will just remove it.

Reply:
After discussion, we have decided to keep it for different PR. Either to reimplement it or remove it. I have added it to my todo list.
Updated PR and will merge after the CI.